lib/mapper.js

```diff
@@ -239,7 +239,8 @@ function customParams(msg, eventName){
       fb_num_items: numberOfItems(msg),
       fb_content_id: contentIds(msg)
   };
-
+  // per FB docs, _appVersion is required in order to populate app version in event dashboard
+  if(msg.obj.context.app.version) customEvents._appVersion = msg.obj.context.app.version;
```

Comment:
@wcjohnson11  Ok but if I include it in the customEvents object without the if statement, then we will have to change a bunch of tests because every track event will contain _appVersion in its output. that ok?